integration_tests/integration/signIn.cy.js

```diff
@@ -41,8 +41,6 @@ context('Sign In', () => {
         .then(href => {
           expect(href).to.equal('http://localhost:9091/auth/account-details')
         })
-
-      page.changeLocationLink().should('not.exist')
```

Comment:
the header is no longer dynamic (as served by wiremock) so change caseload will always exist